pkg/cli/environment.go

```diff
@@ -63,6 +67,8 @@ func New() *EnvSettings {
 	env := EnvSettings{
 		namespace:        os.Getenv("HELM_NAMESPACE"),
 		KubeContext:      os.Getenv("HELM_KUBECONTEXT"),
+		KubeToken:        os.Getenv("HELM_KUBETOKEN"),
+		APIServer:        os.Getenv("HELM_APISERVER"),
```

Comment:
I wonder if we want the variable to be HELM_KUBEAPISERVER?

I'll let @bacongobbler decide.

Reply:
I am updating to KUBEAPISERVER definitely makes more sense